src/transport_performance/gtfs/gtfs_utils.py

```diff
@@ -7,20 +7,135 @@
 import os
 import math
 import plotly.graph_objects as go
-from typing import Union
+from typing import Union, TYPE_CHECKING
 import pathlib
 from geopandas import GeoDataFrame
+import warnings
+
+if TYPE_CHECKING:
+    from transport_performance.gtfs.validation import GtfsInstance
 
 from transport_performance.utils.defence import (
     _is_expected_filetype,
     _check_iterable,
     _type_defence,
     _validate_datestring,
     _enforce_file_extension,
+    _gtfs_defence,
 )
 from transport_performance.utils.constants import PKG_PATH
 
 
+def _validate_bbox(bbox: list) -> None:
+    """Small function to ensure bbox coords are in the correct order."""
+    _type_defence(bbox, "bbox", list)
```

Comment:
doesn't look like we need this - this only gets executed in `filter_gtfs()` if bbox is a list.

Reply:
To give some insight on this, this was added to avoid a complexity hook.

Reply:
sorry, I mean specifically the _type_defence line.

Reply:
Oh right. Happy to remove this or allow it to remain as it might be useful in the future if we implement this function again. Either way is okay wth me.

Reply:
Going to resolve this. Leaving in type defence for extra measures. Un-noticeable computational expenses and also protects for any future uses.